release-notes/rascal-0-41-x-release-notes.md

```diff
@@ -6,28 +6,32 @@ sidebar_position: 87
 
 In this post we report on the Rascal release 0.41.x
 
-## Release 0.41.0 & 0.41.1 - December, 2025
+## Release 0.41.0, 0.41.1 & 0.41.2 - December, 2025
 
-Welcome to Rascal 0.41.0! This release comes with great improvements in usability (parse error recovery, loading speed)
+Welcome to Rascal 0.41.2! This release comes with great improvements in usability (parse error recovery, loading speed)
 and enormous progress with type-checking and compilation. Numerous additions to the standard library and a _big change_ in the Java language support setup...
 These release notes are organized by major topics and there is a list of smaller improvements at the end, including a list of linked closed issues and the merged pull requests.
 
 Many, if not most, of the improvements to the Rascal project were both funded and executed by Swat.engineering BV. Thanks!
 
 :::info
 All Eclipse functionality, including the `rascal-eclipse` plugin and the Eclipse IDE Metatooling Platform (IMP a.k.a. `impulse`), was _archived_ this year, after having been deprecated for a long time.
-Everybody is expected to use Rascal now using the VScode extension, or using the commandline REPL, or from their own LSP clients. With this move to VScode the `Figure` library (embedded in `rascal-eclipse`)
+Everybody is expected to use Rascal now using the VS Code extension, or using the commandline REPL, or from their own LSP clients. With this move to VS Code the `Figure` library (embedded in `rascal-eclipse`)
 is no longer available, until we create a replacement. The pre-existing releases of `rascal-eclipse` and `impulse` will _not_ remain available forever on `usethesource.io`,
 for the sake of security and simplicity.
 :::
 
+### For people using 0.41.0 & 0.41.1
+
+We previously released 0.41.0 and 0.41.1. Since they still contained some issues, we did not release them as part of any Rascal tools (e.g. the VS Code extension). If your project depends on one of those Rascal versions, we urge you to update to 0.41.2.
+
```

Comment:
Let's move this down a bit, lower in the changelog. and maybe change the title to "What happened to 0.41.0 and 0.41.1?" or something?